recipes-kernel/kernel-modules/kernel-module-isp-vvcam_4.2.2.19.0.bb

```diff
@@ -10,6 +10,14 @@ SRCBRANCH = "lf-5.15.y_2.1.0"
 SRCREV = "b26ee8a5402afd7488716e06d82147669c05eb4a"
 
 S = "${WORKDIR}/git/vvcam/v4l2"
+DEBUG_PREFIX_MAP = "-fmacro-prefix-map=${WORKDIR}/git/vvcam=/usr/src/debug/${PN}/${EXTENDPE}${PV}-${PR} \
+ -fdebug-prefix-map=${WORKDIR}/git/vvcam=/usr/src/debug/${PN}/${EXTENDPE}${PV}-${PR} \
+ -fmacro-prefix-map=${B}=/usr/src/debug/${PN}/${EXTENDPE}${PV}-${PR} \
+ -fdebug-prefix-map=${B}=/usr/src/debug/${PN}/${EXTENDPE}${PV}-${PR} \
+ -fdebug-prefix-map=${STAGING_DIR_HOST}= \
+ -fmacro-prefix-map=${STAGING_DIR_HOST}= \
+ -fdebug-prefix-map=${STAGING_DIR_NATIVE}= \
+"
```

Comment:
Hi @MaxKrummenacher, did you consider using `:prepend`?

```
DEBUG_PREFIX_MAP:prepend = " \
    -fmacro-prefix-map=${WORKDIR}/git/vvcam=/usr/src/debug/${PN}/${EXTENDPE}${PV}-${PR} \
    -fdebug-prefix-map=${WORKDIR}/git/vvcam=/usr/src/debug/${PN}/${EXTENDPE}${PV}-${PR} "
```

Better yet would be to tweak the build structure to remove the `../`, right?

Reply:
Hi Tom @thochstein 
I didn't think of doing it with :prepend or :append. Using them likely would be the more future proof solution, you are right.

Even better would be to get rid of the `../`, but I guess that should be done in the upstream project rather than as a patch in a recipe.